odata-json-format/4 Common Characteristics.md

```diff
@@ -417,23 +417,22 @@ if it does not match convention for the localized key values. If the
 `id` is represented, it MAY be a [relative
 URL](#RelativeURLs).
 
-If the entity is transient (i.e. cannot be read or updated), the
+If the entity is transient (see [OData-Protocol](#ODataProtocol)), the
 `id` control information MUST appear in OData 4.0 payloads
-and have the `null` value. In 4.01 payloads transient
-entities need not have the `id` control information, and 4.01
+and have the `null` value. In 4.01 or greater payloads transient
+entities need not have the `id` control information, and
 clients MUST treat entities with neither `id` control
```

Comment:
```suggestion
clients receiving such payloads MUST treat entities with neither `id` control
```
If a 4.0 client receives `"@id": null` and a partial key only, is it bound by this statement?